terraform/main.tf

```diff
@@ -136,8 +136,8 @@ locals {
     prod = {
       vm_size_binarycache     = "Standard_D2_v3"
       osdisk_size_binarycache = "250"
-      vm_size_builder_x86     = "Standard_D16_v3"
-      vm_size_builder_aarch64 = "Standard_D16ps_v5"
+      vm_size_builder_x86     = "Standard_D64_v3"
+      vm_size_builder_aarch64 = "Standard_D64ps_v5"
       osdisk_size_builder     = "500"
       vm_size_controller      = "Standard_E4_v5"
```

Comment:
While we are changing this, let's increase the jenkins-controller size also on the prod. Maybe to "Standard_E8_v5" or even "Standard_E16_v5". Nix evaluation happens on the controller, so I assume growing its size would also help.

Reply:
Increased to 16 core